src/components/SlateEditor/plugins/bodybox/index.js

```diff
@@ -19,12 +20,36 @@ export const defaultBodyBoxBlock = () =>
 
 export default function createBodyBox() {
   const schema = {
-    nodes: {
-      bodybox: SlateBodyBox,
-    },
+    document: {},
+  };
+
+  // Rule to always insert a paragraph as the last node inside if void type
+  function validateNode(node) {
+    if (node.kind !== 'block') return null;
```

Comment:
Kunne vi laget funksjoner som gjør disse tingene? 

For eksempel `nodeIsBlock(node)` og `nodeIsType(node, type)`